diy-trackers-guide.md

```diff
@@ -22,11 +22,32 @@ For more information on what these tracking options look like, please refer to t
 <iframe width="100%" height="auto" src="https://www.youtube.com/embed/Nl_6eQV32ys" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
+## What is an extension
+
+An extension is a singular auxiliary IMU attached to a primary tracker and placed at another location. This allows you to build a secondary tracker without the need for an extra battery, charge board and microcontroller. 
+
+![img](https://i.imgur.com/OxED2eX.png)<br>
+*Example extension build by Rames The Generic#3540*
+
+Due to issues with the extension's cord length we recommend the extension being no further than 45cm from your primary tracker. 
```

Comment:
```suggestion
Due to issues with the extension's cord length, we recommend the extension being no further than 45cm from your microcontroller. 
```

Reply:
I think primary tracker here is better for less tech savy users. Primary tracker makes it very clear in comparison